src/sidebar/components/tag-editor.js

```diff
@@ -115,12 +150,16 @@ function TagEditor({ onEditTags, tags: tagsService, tagList }) {
     setSuggestionsListOpen(false);
     setActiveItem(-1);
 
-    inputEl.current.value = '';
-    inputEl.current.focus();
+    if (inputEl.current) {
```

Comment:
Per my comment above, I think it is OK to skip the `if` check and override the null-ness if we know the value can't actually be null.

Reply:
yep, i agree